tool-plugins/intellij/build.gradle

```diff
@@ -40,11 +40,24 @@ repositories {
     maven { url 'http://dl.bintray.com/jetbrains/intellij-plugin-service' }
 }
 
-apply plugin: 'java'
+//LSP Dependencies
```

Comment:
Add a space after `//` and end with a full stop.

Reply:
Fixed